modules/local/kraken2_standard_report.nf

```diff
@@ -5,7 +5,7 @@ process KRAKEN2_STANDARD_REPORT {
     conda "conda-forge::sed=4.7"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
-        'ubuntu:20.04' }"
+        'docker.io/ubuntu:20.04' }"
```

Comment:
we have a mirror of it on quay.io: https://quay.io/repository/nf-core/ubuntu

Reply:
@jfy133 should I update the modules, or can we fix this in the next release?

Reply:
Communicated yestday on slack: we will update everything for this release

Reply:
Fixed on https://github.com/nf-core/taxprofiler/pull/372